### PR TITLE
Fix for tfsvc artifact

### PR DESCRIPTION
**Pull request by ankitgo**

### CODE REVIEW

File: src/Agent.Worker/Release/Artifacts/TfsVCArtifact.cs [hunk 1]
```diff
@@ -88,8 +91,7 @@ public IArtifactDetails GetArtifactDetails(IExecutionContext context, AgentArtif
                 mappings = JsonConvert.SerializeObject(defaultMapping);
             }
 
-            if (artifactDetails.TryGetValue(ArtifactDefinitionConstants.ProjectId, out projectId)
-                && artifactDetails.TryGetValue(ArtifactDefinitionConstants.RepositoryId, out repositoryId))
+            if (mandatoryFields)
```
Comment (TingluoHuang) [inline src/Agent.Worker/Release/Artifacts/TfsVCArtifact.cs]: What changed, just add a local variable?

Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/TfsVCArtifact.cs]: projectId is being referenced was line no. 68. We did not initialize this variable before as the code that initializes the variable was in line 91. Fix is to move the block of code that initializes the variable above.

Comment (ankitgo) [inline src/Agent.Worker/Release/Artifacts/TfsVCArtifact.cs]: Thanks Sriram..

**Comment by ankitgo** [comment]
Merging this.

**Pull request merged**